source/fundamentals/run-command.txt

```diff
@@ -55,17 +55,46 @@ the current member's role in the replica set, on a database:
 For a full list of database commands and corresponding parameters, see
 the :ref:`Additional Information section <addl-info-runcommand>`.
 
+.. _node-command-options:
+
+Command Options
+---------------
+
+You can set options when executing a database command by
+passing an options object to either command method. The ``command()`` method
+takes a ``RunCommandOptions`` object, and the ``runCursorCommand()`` method
+takes a ``RunCursorCommandOptions`` object.
```

Comment:
Suggestion:

1) I think it would be more clear to specify the command methods rather than expecting the reader to associate it with content from the middle of the previous section.

2) I think "set option ... by passing an options object" is redundant.

3) I think the association between "options" object and the two types of options objects could be more clear.


```suggestion
You can specify optional command behavior for either the ``command()`` or the ``runCursorCommand()`` method.

The ``command()`` method accepts a RunCommandOptions object. To learn more about the RunCommandOptions type, see the API documentation.

The ``runCursor() method`` accepts a ``runCursorCommand()`` object. To learn more about the RunCursorCommandOptions type, see the API documentation.
```